mem/mem.c

```diff
@@ -1129,8 +1129,10 @@ char *comdb2_strndup_static(int indx, const char *s, size_t n)
 
 struct mallinfo comdb2_mallinfo_static(int indx)
 {
+#ifdef INSECURE
     struct mallinfo empty = {0};
     STATIC_RANGE_CHECK(indx, empty);
```

Comment:
`STATIC_RANGE_CHECK(indx, (struct mallinfo){0});` would read neater than this.

Reply:
Should that be Unsecure?

Reply:
That seems to work, thanks Rivers.

Reply:
INSECURE means something else, no?

Reply:
STATIC_RANGE_CHECK is wrapped in an ifdef INSECURE   block. I was using same, but not needed if I use Rivers' trick. Thanks.